ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,458 +1,470 @@
-{%          for network in router_bgp.address_family_ipv4.networks | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_ipv4.networks[network].route_map is defined and router_bgp.address_family_ipv4.networks[network].route_map is not none %}
+{%         for network in router_bgp.address_family_ipv4.networks | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_ipv4.networks[network].route_map is arista.avd.defined %}
       network {{ network }} route-map {{ router_bgp.address_family_ipv4.networks[network].route_map }}
 {%             else %}
       network {{ network }}
 {%             endif %}
-{%          endfor %}
-{%       endif %}
-{%       if router_bgp.address_family_ipv4.peer_groups is defined and router_bgp.address_family_ipv4.peer_groups is not none %}
-{%          for peer_group in router_bgp.address_family_ipv4.peer_groups | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_in is defined and router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_in is not none %}
+{%         endfor %}
+{%         for peer_group in router_bgp.address_family_ipv4.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_in is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_out is defined and router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_out is not none %}
+{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_out is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_ipv4.peer_groups[peer_group].route_map_out }} out
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_in is defined and router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_in is not none %}
+{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_in is arista.avd.defined %}
       neighbor {{ peer_group }} prefix-list {{ router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_out is defined and router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_out is not none %}
+{%             if router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_out is arista.avd.defined %}
       neighbor {{ peer_group }} prefix-list {{ router_bgp.address_family_ipv4.peer_groups[peer_group].prefix_list_out }} out
 {%             endif %}
 {%             if router_bgp.address_family_ipv4.peer_groups[peer_group].activate == true %}
```

Comment:
No check if `router_bgp.address_family_ipv4.peer_groups[peer_group].activate` is defined